src/pages/profile-page/avatar/avatar.js

```diff
@@ -33,42 +32,57 @@ const Avatar = ({ setUserImageUrl, userImageUrl, setUpload, setDeleteAvatar, t }
     return false;
   };
 
-  const checkDimension = async (result) => {
-    const image = new Image();
-
-    const promiseImg = new Promise((resolve, reject) => {
-      image.onload = () => {
-        if (image.height <= 100 && image.width <= 100) {
-          resolve(true);
-        } else {
-          resolve(false);
+  const checkOrResizeImageTo100x100 = (base64Str, maxWidth = 200, maxHeight = 200) => new Promise((resolve) => {
```

Comment:
General functionality that you can reuse should be moved to utils folders.

Reply:
general functions have been moved to utils folder.